docs/getting_started.rst

```diff
@@ -86,6 +117,13 @@ You should now be able to install LXDock using:
 
   $ pip3 install lxdock
 
+.. note::
```

Comment:
It would be nice if we could provide Snap packages and maybe debs or flatpak eventually.  It would make the setup instructions a lot simpler, and this manual setup part can be moved down then.

Reply:
Also this would avoid the issues of lxdock not being available after activating a virtualenv/conda env.